docs/contributing.md

```diff
@@ -48,11 +48,64 @@ These should all be run from within the project directory.
 
 ## Development
 
-### Before You Start
+### Before you start: commit guidelines
 
-Make sure you understand the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification. **All pull requests opened into `main` must have a title that follows the [conventional commits spec](https://github.com/conventional-changelog/commitlint/tree/master/%40commitlint/config-conventional)** This generates an automated changelog entry and is later used to determine versioning for release. It is required to merge.
+Make sure you understand the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification. **All pull requests opened into `main` must have a title that follows the [conventional commits spec](https://github.com/conventional-changelog/commitlint/tree/master/%40commitlint/config-conventional)**. This generates an automated changelog entry and is later used to determine versioning for release. It is required to merge.
 
-### Pull Requests
+The format for PR titles is:
+
+```
+<type>(scope): <subject>
+<BLANK LINE>
+<body>
+<BLANK LINE>
+<footer>
+```
+
+#### `type`:
+
```

Comment:
Should we also include `!` or `BREAKING_CHANGE` in the PR title as well if its a breaking change since we squash

Reply:
Also this level of detail for contributors feels like a lot.  Makes me wish for something like the form described [here](https://www.conventionalcommits.org/en/v1.0.0-beta.4/#do-all-my-contributors-need-to-use-the-conventional-commit-specification)  so maintainers could carry the bulk of this work.

Reply:
I agree, although since only Trussels are able to merge PRs right now maybe it's okay for now? definitely something to think about moving forward..